tests/unit/specs/components/forms/DocumentUploadForm/__snapshots__/UploadError.spec.js.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DocumentUploadForm > UploadError render should correctly render Upload Error component 1`] = `
+<div>
+  <upload-title-stub title="Error" iserror="true"></upload-title-stub>
```

Comment:
Because we should check error case and case without error

Reply:
not sure what do you mean to check here, attrs? or something else?

Reply:
It's not render section responsobulity. You should use behavior or props section